sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -98,6 +99,8 @@ class Analyzer(
     this(catalog, conf, conf.optimizerMaxIterations)
   }
 
+  private lazy val random = new Random()
```

Comment:
Shall we put `random` in the `ResolvedUuidExpressions`? That makes it a little bit easier to follow.